test/zilliqa.live.test.ts

```diff
@@ -1,35 +1,62 @@
 import { BN } from "@zilliqa-js/util";
 import { expect } from "chai";
 
-import { initZilliqa } from "../src/ScillaContractDeployer";
 import * as ZilliqaHardhatObject from "../src/ZilliqaHardhatObject";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { useEnvironment } from "./helpers"
 
 describe("", function () {
-  let zobj: ZilliqaHardhatObject.ZilliqaHardhatObject;
 
-  before(async function () {
-    // const privateKeys = [ "d7ebc171562928a59aa8423e9b69393fe43a32f34b25dddc04f3f0dfe8881479" ];
-    const privateKeys = [
-      "603f0ef1610a638bb28d8f61ed7956c701e114fa39e1c263fc4f5504aed2f211",
-    ];
-    // const network_url = "http://localhost:8082";
-    const network_url =
-      process.env.ZILLIQA_API_URL || "https://dev-api.zilliqa.com";
-    const chain_id = 333;
-
-    await initZilliqa(network_url, chain_id, privateKeys);
-    zobj = new ZilliqaHardhatObject.ZilliqaHardhatObject();
-  });
+  var hre : HardhatRuntimeEnvironment;
+  var zobj : ZilliqaHardhatObject.ZilliqaHardhatObject;
 
+  useEnvironment("hardhat-project");
+  this.timeout(500000);
   describe("Zilliqa network APIs", function () {
-    // I happen to know that this account exists - rrw 2023-03-12
-    it("Should be able to fetch a balance", async function () {
-      const account = zobj.getAccounts()[0];
-      const [bal, nonce] = await zobj.getBalance(account);
+    // Try an account that (hopefully) doesn't exist.
+    it("Should be able to fetch a balance from a nonexistent account", async function () {
+      const privKey = this.zobj.createPrivateKey();
+      const [ acc, idx ] = this.zobj.pushPrivateKey(privKey);
+      //console.log(`idx ${idx}`);
```

Comment:
Can we remote this commented line?

Reply:
and all commented ones? :)

Reply:
OK, but they're in a test - it's not as though they're a big maintenance burden, and rewriting them all when something fails and you need to debug stuff is really tedious....